From 3b8ef584594048514645bc818ab5dd43cf882fe1 Mon Sep 17 00:00:00 2001
From: Paul Lemire <paul.lemire@kdab.com>
Date: Mon, 4 Jan 2021 07:21:14 +0100
Subject: [PATCH 19/19] qpicktriangleevent_p.h Add missing private API warning

Change-Id: I57f758f279d62ee4508914d12f99ee68cd41a2c7
Reviewed-by: Mike Krus <mike.krus@kdab.com>
(cherry picked from commit 45dfb8a03f4ee2abb963ce3726eaef0dd1c2c521)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/render/picking/qpicktriangleevent_p.h | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/src/render/picking/qpicktriangleevent_p.h b/src/render/picking/qpicktriangleevent_p.h
index d646ac2bb..8ed4f26f5 100644
--- a/src/render/picking/qpicktriangleevent_p.h
+++ b/src/render/picking/qpicktriangleevent_p.h
@@ -40,6 +40,17 @@
 #ifndef QT3DRENDER_QPICKTRIANGLEEVENT_P_H
 #define QT3DRENDER_QPICKTRIANGLEEVENT_P_H
 
+//
+//  W A R N I N G
+//  -------------
+//
+// This file is not part of the Qt API.  It exists for the convenience
+// of other Qt classes.  This header file may change from version to
+// version without notice, or even be removed.
+//
+// We mean it.
+//
+
 #include <Qt3DRender/private/qpickevent_p.h>
 
 QT_BEGIN_NAMESPACE
-- 
2.31.1

